ChatSDKSamples/ChatSDK_Auth/ChatSDK_Auth/ZendeskMessaging.swift

```diff
@@ -25,7 +25,7 @@ final class ZendeskMessaging: NSObject, JWTAuthenticator {
     }
 
     #warning("Please provide Chat account key")
-    let accountKey = "<#String#>"
+    let accountKey = "OXgeUeElrZ7hzQ96yrELV4xxlJC91awA"
```

Comment:
shall we use some key or a placeholder here?

Reply:
I think it's fine to leave a placeholder for customers to provide their own key here